src/apps/routes/schemas/app.clj

```diff
@@ -261,8 +262,9 @@
    :name (describe String "The App Category's name")})
 
 (defschema AppDetailsTool
-  (assoc Tool
-    :id (describe String "The tool identifier.")))
+  (merge Tool
+         {:id                       (describe String "The tool identifier.")
+          (optional-key :container) {:image (dissoc containers/Image :id)}}))
```

Comment:
It looks like the same `:image` schema is used in the [ToolListingItem](https://github.com/cyverse-de/apps/blob/master/src/apps/routes/schemas/tool.clj#L65).
Maybe this could be refactored into a common schema in `apps.routes.schemas.tool`?

It's also pretty simple and only used in 2 places so far, so I don't think it should be required for this PR.